tests/test_registry.py

```diff
@@ -119,6 +125,265 @@ def test_registrations(self):
         eq_(registry, registration.registry)
         eq_(self._default_library, registration.library)
 
+    def test_fetch_catalog(self):
```

Comment:
This code existed before, in RemoteRegistry, but was not tested. Since I moved it to another class I decided it was a good opportunity to add test coverage.